src/Cms/File.php

```diff
@@ -228,6 +228,28 @@ public function contentFileName(): string
         return $this->filename();
     }
 
+    /**
+     * Path to content lock file
+     *
+     * @internal
+     * @return string
+     */
+    public function contentLockFile(): string
```

Comment:
Are those two methods left-overs from the old implementation?

Reply:
Indeed.